pkg/api/v1beta1/types.go

```diff
@@ -1522,3 +1526,27 @@ type GlusterfsVolumeSource struct {
 	// the Glusterfs volume to be mounted with read-only permissions
 	ReadOnly bool `json:"readOnly,omitempty" description:"Glusterfs volume to be mounted with read-only permissions"`
 }
+
+// RBDVolumeSource represents a Rados Block Device Mount that lasts the lifetime of a pod
+type RBDVolumeSource struct {
+	// Required: EndpointsName is the endpoint name that represents a collection of Ceph monitors
+	EndpointsName string `json:"endpoints" description:"a collection of Ceph monitors"`
```

Comment:
I'm not super concerned about API compat with gluster - we could PROBABLY take the hit and just break compat.  Or make a new field that is an ObjectRef, deprecate the old field, but use it preferentially if specified.  That's not too hard, just need to pick a name that you don't hate, and be consistent across all of these scenarios.